tests/v2/test_waku_switch.nim

```diff
@@ -100,3 +111,37 @@ suite "Waku Switch":
 
     ## Teardown
     await allFutures(wakuSwitch.stop(), sourceSwitch.stop(), destSwitch.stop())
+
+  asyncTest "Waku Switch uses Rendezvous":
```

Comment:
I wouldn't include this in the `switch` tests. Since this is a separate protocol I suggest creating a new `test_waku_rendezvous` suite (even if it contains only the one test for now).